flight/Modules/ManualControl/failsafe_control.c

```diff
@@ -77,7 +77,7 @@ int32_t failsafe_control_select(bool reset_controller)
 	SystemSettingsAirframeTypeGet(&airframe_type);
 
 	StabilizationDesiredData stabilization_desired;
-	stabilization_desired.Thrust = (airframe_type == SYSTEMSETTINGS_AIRFRAMETYPE_HELICP) ? 0 : -1;
+	stabilization_desired.Thrust = 0;
```

Comment:
This is now consistent between helis and the rest of aircraft-- no longer is there a "special region" of -1..0 for multirotors.